### ISSUE TITLE
Can I use fields if not exists in schema to solr?

### ISSUE THREAD

**Issue by suhyunjeon** [lucidworks/banana#126]
label:question 
I want to use a field does not exist in schema from banana.  

1) Added Add Custom Query in Queries panel like this.  

```
&fq=extra_field":if(exists(tnx_abnormal_b),0,1)   
```

2) Selected table to the term panel. Added Field "extra_field" in Panel tab.  
3) Got the bug  

```
Cannot read the field in schema.  
```

Can I use fields if not exists in schema to solr?

**Comment by aadel** [comment]
Yes, dynamic fields are supported in Banana. It could be an issue with custom query syntax, maybe it should be `&fq=extra_field:"if(exists(tnx_abnormal_b),0,1)"`
